src/Polly.Core/CircuitBreaker/OnCircuitClosedArguments.cs

```diff
@@ -5,6 +5,9 @@ namespace Polly.CircuitBreaker;
 /// <summary>
 /// Arguments used by <see cref="CircuitBreakerStrategyOptions{TResult}.OnClosed"/> event.
 /// </summary>
+/// <remarks>
+/// Always use constructor when creating this struct, otherwise we do not guarantee the binary compatibility.
```

Comment:
Nit on these comments:

```suggestion
/// Always use the constructor when creating this struct, otherwise we do not guarantee binary compatibility.
```